src/geometry/prototype/RPC.cc

```diff
@@ -37,50 +38,45 @@ const RPC::RPCInfo RPC::rpc_infos[n_rpcs] = {
   { 120.28*mm,  616.35*mm,  1423.12*mm,  1.37958*rad},
   {-147.19*mm, -601.52*mm,  1323.12*mm,  1.38013*rad}
 };
+//----------------------------------------------------------------------------------------------
 
 //__RPC Pad Constructor_________________________________________________________________________
 RPC::Pad::Pad(int input_id) : id(input_id) {}
 //----------------------------------------------------------------------------------------------
 
 //__RPC Constructor_____________________________________________________________________________
 RPC::RPC(int input_id) : _pads(), _id(input_id), _name("RPC" + std::to_string(1 + input_id)) {
-  _volume = Construction::OpenBoxVolume(
-    _name,
-    Width, Height, Depth, Thickness,
-    Material::Casing);
+  _volume = Construction::BoxVolume(_name, Width, Height, Depth, Material::Casing);
```

Comment:
I'm a bit concerned about changing to a BoxVolume here. Doesn't that make the RPC a solid block of aluminum? The aluminum casing is only about 1 mm thick.

Reply:
It doesn't actually make it a block of aluminum because the gas overrides the volume when inserting. I wanted to do it with a casing (to be more correct) but then it won't propagate the particles through the material at all.